ReportingServicesTools.psd1

```diff
@@ -7,7 +7,7 @@
 	RootModule = 'ReportingServicesTools.psm1'
 	
 	# Version number of this module.
-	ModuleVersion = '0.0.0.8'
+	ModuleVersion = '0.1.0.9'
```

Comment:
Module version gets populated automatically when we use the CI process with a tag to publish to the Pshell Gallery, you can leave it unmodified